styles/utility.css

```diff
@@ -92,13 +114,21 @@ code {
   display: flex;
   align-items: center;
 }
-.mt-2 {
-  margin-top: 2rem;
-}
-.mb-2 {
-  margin-bottom: 2rem;
-}
+
 .link-secondary-color {
   text-decoration: none;
   color: var(--secondary-color);
 }
+.border-2 {
+  border: 2px solid var(--primary-color);
+}
+.border-primary {
+  border-color: var(--primary-color);
+}
+.border-solid {
+  border-style: solid;
+}
+
+.border-green-600 {
+  border-color: rgb(22 163 74);
+}
```

Comment:
add EOF line here.

Reply:
EOF is showing in VS Code, but somehow not showing here.
Tried adding it but prettier is removing that.